lib/helpers.js

```diff
@@ -171,11 +171,17 @@ export function getTempDir(prefix = '') {
         return;
       }
 
-      fs.realpath(folder, (realError, rpath) => (realError ? reject(realError) : resolve(rpath)));
+      resolve(realPath(folder));
     });
   });
 }
 
+export function realPath(folder) {
+  return new Promise((resolve, reject) => {
+    fs.realpath(folder, (err, rpath) => (err ? reject(err) : resolve(rpath)));
+  });
+}
+
```

Comment:
:+1: Excellent, this will be handy.

As a sidenote - how would we feel about using something like [a recent `fs-extra`](https://www.npmjs.com/package/fs-extra) or [`mz`](https://www.npmjs.com/package/mz) to handle promisified filesystem operations? It'd cut down on the number of helpers we need here. But then, maybe we don't need that many.